website_analytics_matomo/views/templates.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <template id="layout" inherit_id="website.layout">
+        <xpath expr="//script[@id='tracking_code']" position="after">
+            <t
+                t-if="website and website.has_matomo_analytics and website.matomo_analytics_host"
+            >
+                <script type="text/javascript">
+                    var _paq = _paq || [];
+                    _paq.push(['trackPageView']);
+                    _paq.push(['enableLinkTracking']);
+                    (function() {
+                    var u="//<t t-raw="website.matomo_analytics_host" />/";
+                    _paq.push(['setTrackerUrl', u+'matomo.php']);
+                    _paq.push(['setSiteId', <t t-raw="website.matomo_analytics_id" />]);
```

Comment:
Since you converted the `matomo_analytics_id` to Char, now the apex are probably needed:

```suggestion
                    _paq.push(['setSiteId', '<t t-raw="website.matomo_analytics_id" />']);
```